src/main/resources/de/qabel/desktop/ui/accounting/login/login.fxml

```diff
@@ -5,82 +5,109 @@
+						<Insets left="10.0" right="10.0"/>
                     </VBox.margin>
                 </TextField>
 
-                <TextField fx:id="email" alignment="CENTER" managed="false" promptText="%loginUserEMail" styleClass="user">
+                <TextField fx:id="email" alignment="CENTER" managed="false" prefHeight="26.0" promptText="%loginUserEMail" styleClass="user">
                     <VBox.margin>
-                        <Insets/>
+						<Insets left="10.0" right="10.0"/>
                     </VBox.margin>
                 </TextField>
 
-                <PasswordField fx:id="password" alignment="CENTER" promptText="%loginBoxAccountUserPassword" text="testuser">
+				<PasswordField fx:id="password" alignment="CENTER" prefHeight="26.0" prefWidth="220.0"
+							   promptText="%loginBoxAccountUserPassword" text="testuser">
                     <VBox.margin>
-                        <Insets/>
+						<Insets left="10.0" right="10.0"/>
                     </VBox.margin>
                 </PasswordField>
 
-                <PasswordField fx:id="confirm" alignment="CENTER" managed="false" promptText="%loginConfirmPassword">
+                <PasswordField fx:id="confirm" alignment="CENTER" managed="false" prefHeight="26.0" promptText="%loginConfirmPassword">
                     <VBox.margin>
-                        <Insets/>
+						<Insets left="10.0" right="10.0"/>
                     </VBox.margin>
                 </PasswordField>
 
-                <HBox fx:id="buttonBar" alignment="CENTER" styleClass="login-button-bar">
+				<HBox fx:id="buttonBar" alignment="CENTER" prefHeight="26.0" prefWidth="220.0"
+					  styleClass="login-button-bar">
                     <children>
-                        <Button fx:id="openCreateButton" mnemonicParsing="false" onAction="#openCreateBoxAccountSetup"
-                                styleClass="create-account-button" text="%loginOpenCreateBoxAccountDialog" managed="true"/>
-                        <Button fx:id="createButton" mnemonicParsing="false" onAction="#createBoxAccount"
-                                styleClass="create-account-button" text="%loginCreateBoxAccount" managed="false"/>
-                        <Button fx:id="loginButton" mnemonicParsing="false" onAction="#login" styleClass="login-button"
-                                text="%loginLogin"/>
-
+						<Button fx:id="openCreateButton" managed="true" mnemonicParsing="false" onAction="#openCreateBoxAccountSetup" prefHeight="26.0" prefWidth="221.0" styleClass="create-account-button" text="%loginOpenCreateBoxAccountDialog">
+							<HBox.margin>
+								<Insets left="10.0" right="10.0"/>
+							</HBox.margin>
+						</Button>
+						<Button fx:id="createButton" managed="false" mnemonicParsing="false"
+								onAction="#createBoxAccount" prefHeight="26.0" prefWidth="221.0"
+								styleClass="create-account-button" text="%loginCreateBoxAccount">
+							<HBox.margin>
+								<Insets left="10.0" right="10.0"/>
```

Comment:
HBox with horizontal inset is spacing (`<HBox spacing="20"` or `-fx-spacing: 20`)